docs/system/charge_controller.md

```diff
@@ -1,9 +1,94 @@
+
+## Solar
+
+Solar energy is the predominant method for off-grid electricity generation, as it does not require any moveable parts and is easily installed.
+PWM (Pulse Width Modulation) charge controller is in pratice a switch connected between solar panel and a battery. During bulk or constant current load, the switch is simply closed and the battery is charged with whatever current the panel can provide. The panel's voltage is pulled down which effectively lowers the panels efficiency since it is not operating in its **maximum power point**. After bulk charging, the pwm-controller will apply different duty cycles resulting in current boosts, effectively applying a constant voltage. The cycles are adapted depending on the the battery's charge state. The **interactive** example in Figure [1] shows the power effectively used for charging a battery with a given base voltage. The closer the rated battery voltage to the maximum power point voltage, the higher the overall efficiency.
+
+As explained in the [Solar Panel](solar_panel.md) chapter, the voltage of a solar panel depends on the number of cells, the temperature, the irradiance and the amount current draw.
+
+Most 12 V solar panels output 16-20 V in their maximum power point, whereas 12 V batteries usually need around 14-14.5 V to get fully charged. Direct connection of batteries to the solar panel without any regulation causes damage to the batteries as a result of overcharging.
+
+There are mainly two different types of charge controllers, the Maximum Power Point Trackers (MPPT) and cheaper pulse-width modulated (PWM) series switch regulators.
+
+### Maximum Power Point Tracker
+
+<fig-caption src="system/mppt-charge-controller.svg" caption="MPPT charge controller" num="1" />
+
+MPPT (Maximum Power Point Tracking) controllers contain a DC-DC converter which matches varying voltage at the solar panel input with the output at the battery side.
+
+Because of the typically high conversion efficiencies (>95%), power at input and output remains the same whereas the voltage and current varies correspondingly. In order to get the best out of solar panels, MPPT charge controller finds and operates at optimal current-voltage point. In contrast, PWM controllers operate at the voltage equal to that of battery voltage which is not an optimal current-voltage point resulting in wastage of power. There are many ways to implement the tracking, two will be described here.
+
+#### Fractional Open Circuit Voltage
+
+This methods uses a simple linear relationship between the open circuit and maximum powerpoint voltage:
+
+$$V_{MPP} = k \cdot V_{oc}$$
+
+The factor $k$ can be determined for a given solar panel than used to track the maximum. It is dependant on the panels' temperature though and has to be corrected accordingly. This method is simple to implement though the factor and its temperature dependency has to be adapted to every installation so that the controller itself is not that versatile.
```

Comment:
```suggestion
The factor $k$ can be determined for a given solar panel and afterwards used to track the maximum. It is dependent on the panel's temperature, though, and has to be corrected accordingly. This method is simple to implement, but the parameters have to be adapted for every installation so that the controller itself is not very versatile.
```